piker/brokers/kraken.py

```diff
@@ -271,18 +454,304 @@ async def bars(
             raise SymbolNotFound(json['error'][0] + f': {symbol}')
 
 
+
 @asynccontextmanager
 async def get_client() -> Client:
     client = Client()
 
+    section = get_config()
+    client._name = section['key_descr']
```

Comment:
This is, interesting. Why aren't these assigned in `.__init__()` which is more traditional style?